PROVIDER_CHECKLIST.md

```diff
@@ -0,0 +1,48 @@
+
+To build a new provider, please refer to the  [Provider Development Guide](https://crossplane.io/docs/master/contributing/provider_development_guide.html).
+
+This document contains list of items that are usually involved in managing Open Source contributions to a Crossplane provider. In general your provider should follow the guidelines documented in the [Crossplane Contributing](https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md) guide.
+
+## Repository
+
+It is recommended to use the [crossplane-contrib](https://github.com/orgs/crossplane-contrib) Github Organization. Your project should be named `provider-<name>`. Members of the Crossplane community are happy to create a repository and configure access. Please reach out to the Crossplane [#dev](https://crossplane.slack.com/archives/CEF5N8X08) channel.
+
+There is a template repository called [provider-template](https://github.com/crossplane/provider-template) that can be used as a starting point for new providers.
+
+## Files
+
+Most Crossplane providers include the following files:
+
+- [ ]  A descriptive README.md at the root of the project (see [provider-gcp/README.md](https://github.com/crossplane/provider-gcp/blob/master/README.md) as an example)
+- [ ]  Code is licensed under the[ Apache 2.0 License](https://github.com/crossplane/provider-template/blob/main/LICENSE)
+- [ ]  Include a “Developer Certificate of Origin”. Example: [DCO](https://github.com/upbound/build/blob/master/DCO) 
+- [ ]  Include the CNCF [Code of Conduct](https://github.com/crossplane/crossplane/blob/master/CODE_OF_CONDUCT.md)
+- [ ]  Update [OWNERS.md](https://github.com/crossplane/provider-template/blob/main/OWNERS.md) with contacts for project Owners
+- [ ]  Ensure `hack/boilerplate.go.txt` (used in Code generation) includes Crossplane Authors, Apache license and any other Copyright statements: [https://github.com/crossplane/provider-aws/blob/master/hack/boilerplate.go.txt](https://github.com/crossplane/provider-aws/blob/master/hack/boilerplate.go.txt) 
+- [ ] Include Documentation on how to:
+  - [ ] Install Provider
+  - [ ] Contribute to Development
+  - [ ] Authenticate to backend API, including creating Kubernetes secrets for the ProviderConfig
+- [ ] Include examples for the ProviderConfig and each resource in the `examples/` directory.
+
+## Build Process
+
+There are a number of build tools and processes that are common across the Crossplane ecosystem. Using these ensures a consistent development environment across projects.
+
+- [ ]  Use the [Upbound build](https://github.com/upbound/build) submodule. (see [https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md#establishing-a-development-environment](https://github.com/crossplane/crossplane/blob/master/CONTRIBUTING.md#establishing-a-development-environment)) 
```

Comment:
https://github.com/crossplane/provider-template/pull/19#issuecomment-987182490

I have an ancient PR that would make this the default. I'll try follow up this week.

Reply:
If you can't get to it, I will be happy to take a look. I've moved a couple of repos to the build submodule recently so it shouldn't take that long.

Reply:
It's all updated now, and I think there's an appetite to merge it. Doing so would let us simplify this section to something like:

1. Use the build submodule.
2. If you're part of the crossplane-contrib org and want to push images to the crossplane org in Docker Hub please ask a steering committee member to grant your project access to the GitHub org scoped secrets.

Reply:
I've updated the text and broke out the artifacts section.